package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "cpmonitor",
+  "packageManager": "yarn@3.4.1",
```

Comment:
Just my newbie question: Any advantage over `npm`, nowadays?

Reply:
ChatGPT says:

Yarn and npm are both package managers for Node.js, which are used to manage dependencies and packages for a project. While they both have similar features, there are a few advantages to using Yarn over npm:

    Faster Performance: Yarn is generally faster than npm because it caches packages locally, allowing for quicker installations and updates. Yarn also uses parallel processing, which allows it to install packages more quickly.

    Deterministic Dependencies: Yarn creates a lock file (yarn.lock) that ensures that all installations are consistent across different machines. This ensures that all team members have the same dependencies installed, which can prevent issues caused by version mismatches.

    Better Security: Yarn has a feature called "yarn audit" that checks for known security vulnerabilities in the packages that you are using. It also supports the use of "yarn resolutions," which allows you to force a specific version of a package to be installed, even if it has known vulnerabilities.

    Better User Experience: Yarn has a more user-friendly CLI (command-line interface) and provides more detailed error messages than npm. It also has a feature called "workspaces," which allows you to manage multiple packages within a single repository more easily.

Reply:
:rofl: ChatGPT is the source of truth, nowadays? Gotta get used to that. It actually quotes some outdated information: I read a few of these points some month ago (perfomance, determinism, audit) and also that by now `npm` caught up with `yarn`. I used `npm` just a little and `yarn` not at all. Therefore, I am probably not a better source than AI. Its fine with me to stick with `yarn`. Whatever you know better.

Reply:
:laughing: thats we I told my source. Sorry I was too lazy to look it up :sweat_smile: